Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM node as build-env
+
+RUN mkdir /work
+
+ADD ./ /work/
+
+WORKDIR /work
+
+RUN npm install && npm run downloadsolc && npm run make-mock-compiler && npm run build
```

Comment:
Is `npm run downloadsolc && npm run make-mock-compiler` necessary? If not, I will delete it.

Reply:
`npm run make-mock-compiler` is not necessary

Reply:
and I think that `npm run downloadsolc_root` should be used instead of `npm run downloadsolc` . https://github.com/ethereum/remix-ide/blob/master/package.json#L148